cli/internal/run/run.go

```diff
@@ -905,62 +965,6 @@ func (e *execContext) exec(pt *nodes.PackageTask, deps dag.Set) error {
 	return nil
 }
 
-func (r *run) generateDotGraph(taskGraph *dag.AcyclicGraph, outputFilename fs.AbsolutePath) error {
-	graphString := string(taskGraph.Dot(&dag.DotOpts{
-		Verbose:    true,
-		DrawCycles: true,
-	}))
-	ext := outputFilename.Ext()
-	if ext == ".html" {
-		f, err := outputFilename.Create()
-		if err != nil {
-			return fmt.Errorf("error writing graph: %w", err)
-		}
-		defer f.Close()
-		f.WriteString(`<!DOCTYPE html>
-    <html>
-    <head>
-      <meta charset="utf-8">
-      <title>Graph</title>
-    </head>
-    <body>
-      <script src="https://cdn.jsdelivr.net/npm/viz.js@2.1.2-pre.1/viz.js"></script>
-      <script src="https://cdn.jsdelivr.net/npm/viz.js@2.1.2-pre.1/full.render.js"></script>
-      <script>`)
-		f.WriteString("const s = `" + graphString + "`.replace(/\\_\\_\\_ROOT\\_\\_\\_/g, \"Root\").replace(/\\[root\\]/g, \"\");new Viz().renderSVGElement(s).then(el => document.body.appendChild(el)).catch(e => console.error(e));")
-		f.WriteString(`
-    </script>
-  </body>
-  </html>`)
-		r.ui.Output("")
-		r.ui.Output(fmt.Sprintf("✔ Generated task graph in %s", ui.Bold(outputFilename.ToString())))
-		if ui.IsTTY {
-			if err := browser.OpenBrowser(outputFilename.ToString()); err != nil {
-				r.ui.Warn(color.New(color.FgYellow, color.Bold, color.ReverseVideo).Sprintf("failed to open browser. Please navigate to file://%v", filepath.ToSlash(outputFilename.ToString())))
-			}
-		}
-		return nil
-	}
-	hasDot := hasGraphViz()
-	if hasDot {
-		dotArgs := []string{"-T" + ext[1:], "-o", outputFilename.ToString()}
-		cmd := exec.Command("dot", dotArgs...)
-		cmd.Stdin = strings.NewReader(graphString)
-		if err := cmd.Run(); err != nil {
-			return fmt.Errorf("could not generate task graphfile %v:  %w", outputFilename, err)
-		} else {
-			r.ui.Output("")
-			r.ui.Output(fmt.Sprintf("✔ Generated task graph in %s", ui.Bold(outputFilename.ToString())))
-		}
-	} else {
-		r.ui.Output("")
-		r.ui.Warn(color.New(color.FgYellow, color.Bold, color.ReverseVideo).Sprint(" WARNING ") + color.YellowString(" `turbo` uses Graphviz to generate an image of your\ngraph, but Graphviz isn't installed on this machine.\n\nYou can download Graphviz from https://graphviz.org/download.\n\nIn the meantime, you can use this string output with an\nonline Dot graph viewer."))
-		r.ui.Output("")
-		r.ui.Output(graphString)
-	}
-	return nil
-}
-
```

Comment:
🎉 The more things that get removed from `run` the better our lives will be. This is great!